sdk/ml/azure-ai-ml/azure/ai/ml/_schema/component/input_output.py

```diff
@@ -96,18 +96,15 @@ class Meta:
         allowed_values=SUPPORTED_PARAM_TYPES,
         required=True,
     )
-    # hide is_control and early_available in spec
+    # hide early_available in spec
     if is_private_preview_enabled():
-        is_control = fields.Bool()
```

Comment:
Does it need to be added here `_is_primitive_type`=True?
I understand that it should not be necessary.

Reply:
we should not as that's not part of yaml (schema).